autoptsclient_common.py

```diff
@@ -503,6 +504,17 @@ def get_test_case_description(cache, test_case_name):
             return test_case.find('description').text
 
 
+def get_the_color(status):
```

Comment:
get_the_color is not easy to understand. I would suggest to change functions name to:

```
get_result_color
get_term_color
get_text_color
```

Or something similar, without 'the' but actually specifying what the color is used for.

Other than that PR looks great

Reply:
Done